tools/scripts/generic.mk

```diff
@@ -379,18 +380,11 @@ clean_all: clean_libs
 	@$(call print,[Delete] $(BUILD_DIR))
 	$(MUTE) $(call remove_dir,$(BUILD_DIR)) $(HIDE)
 
-PHONY += ca
```

Comment:
I would rephrase the commit title to "...: remove obsolete/unneeded rules" or something like that

Reply:
Done